aspnet/migration/rc1-to-rc2.rst

```diff
@@ -180,36 +97,31 @@ WebApplicationConfiguration    WebHostConfiguration
 
 The ``commands`` section of ``project.json`` has been removed completely. Use ``dotnet run`` or ``dotnet <dllname>`` instead.
 
+Hosting Configuration 
+---------------------
+You enable configuration for your web application host by calling the ``UseConfiguration()`` method on ``WebHostBuilder`` and passing a configuration instance. The following configuration values can be specified:
 
-IHostingEnvironment changes 
----------------------------
+-application
+-startupAssembly
+-detailedErrors
+-server
+-webroot
+-captureStartupErrors
+-server.urls
+-contentRoot
 
-All environment variables are now prefixed with the ``ASPNETCORE_`` prefix.
+To configure hosting using environmental variables, you must add them as a configuration source, and optionally specify a prefix.
```

Comment:
> To configure hosting using environmental variables, you must add them as a configuration source, and optionally specify a prefix.

To configure hosting using environmental variables, you must add them as a configuration source, and optionally specify a prefix. In RC2, you can use whatever prefix you want. You should add it explicitly by calling: